### PR TITLE
SAIL address and move company records test

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        this.lookUpROAddress(buildingNameNumber, postcode);
+        cy.accessibilityCheck();
+        cy.get('#company-address-container-continue').click();
+    }
+
+    changeLocationOfRegisters() {
+        cy.get('#sail-registers-container-change').click();
+        cy.get('#checkRegsAll').click();
+        cy.accessibilityCheck();
+        cy.get('li[id*=D_I_R_I_N_D_E_M__d_i_s_p_l_a_y_-selection').click();
```
Comment (rsuller) [inline cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js]: These selectors 😆

File: cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import AddressPage from "./generic/Address"
+
+//const addressPage = new AddressPage();
```
Comment (rsuller) [inline cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js]: Required?

Comment (danfalc) [reply cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js]: removed

File: cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js (hunk 1 above)
Comment (danfalc) [reply cypress/support/page_objects/NotificationOfSailAndRegisterLocationPage.js]: Insane!!!

**Pull request merged**